lib/improver/tests/acceptance/test_neighbour_finding.py

```diff
@@ -84,8 +88,11 @@ def test_nearest_minimum_dz(tmp_path, domain, model):
     orography_path = kgo_dir / f"inputs/{model}_orography.nc"
     landmask_path = kgo_dir / f"inputs/{model}_landmask.nc"
     output_path = tmp_path / "output.nc"
-    args = [sites_path, orography_path, landmask_path, output_path,
-            "--minimum_dz", "--search_radius", "50000"]
+    args = [orography_path, landmask_path,
+            "--site-list", sites_path,
+            "--minimum-dz",
+             "--search-radius", "50000",
```

Comment:
Bonus extra space indentation.
```suggestion
            "--search-radius", "50000",
```